ethereum_history_api/circuits/get_receipt/src/main.nr

```diff
@@ -0,0 +1,20 @@
+use dep::ethereum_history_api::receipt::{get_receipt, TxReceiptWithinBlock, FakeTurbofishReceiptProofLen, FakeTurbofishReceiptRlpLen};
+
+global LOG_NUM = 1;
+global MAX_LOG_DATA_SIZE = 128; // 4 * 32 (Eth word length)
+global MAX_RECEIPT_PROOF_LENGTH = 3090; // MAX_TRIE_NODE_LENGTH * MAX_RECEIPT_TREE_DEPTH
+global MAX_RECEIPT_RLP_LENGTH = 512; // TODO: Put the real value here
+
+fn main(
+    block_number: pub Field,
+    tx_idx: pub Field
+) -> pub TxReceiptWithinBlock<LOG_NUM, MAX_LOG_DATA_SIZE> {
+    let max_receipt_proof_len: FakeTurbofishReceiptProofLen<MAX_RECEIPT_PROOF_LENGTH> = FakeTurbofishReceiptProofLen {};
+    let max_receipt_rlp_length: FakeTurbofishReceiptRlpLen<MAX_RECEIPT_RLP_LENGTH> = FakeTurbofishReceiptRlpLen {};
```

Comment:
I propose `PhantomReceiptRlpLen`

https://doc.rust-lang.org/rust-by-example/generics/phantom.html

Reply:
Like it. Done!